HabitRPG/HRPGModels/ChallengeTask+CoreDataClass.m

```diff
@@ -8,6 +8,46 @@
 
 #import "ChallengeTask+CoreDataClass.h"
 #import "Challenge+CoreDataClass.h"
+#import "UIColor+Habitica.h"
+
 @implementation ChallengeTask
 
+- (UIColor *)taskColor {
```

Comment:
considering this is now duplicated across two classes, we should probably move it somewhere else. maybe static methods on a helper class? Or add them to the UIColor extension?

Reply:
Let me see what I can do!